src/utils/__tests__/Browser.test.js

```diff
@@ -34,6 +34,24 @@ describe('util/Browser/isIE()', () => {
     });
 });
 
+describe('util/Browser/isSafari()', () => {
+    test('should return true if Safari', () => {
+        browser.getUserAgent = jest.fn().mockReturnValueOnce('AppleWebKit/4.0');
+        expect(browser.isSafari()).toBeTruthy();
+    });
+    test('should return false if not Safari', () => {
+        browser.getUserAgent = jest.fn().mockReturnValueOnce('Trident');
+        expect(browser.isSafari()).toBeFalsy();
+    });
+
+    test('should not incorrectly identify Blink as WebKit', () => {
+        browser.getUserAgent = jest
+            .fn()
+            .mockReturnValueOnce('AppleWebKit/7.8 (KHTML, like Gecko) Chrome/1.2.3.4 Safari/5.6');
+        expect(browser.isSafari()).toBeFalsy();
```

Comment:
Would it make sense to convert this to a `test.each` to avoid the code repetition?